modules/support-reviewing-an-access-request-from-an-email-notification.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-support.adoc
+// * support/getting-support.adoc
+// * service_mesh/v2x/ossm-troubleshooting-istio.adoc
+// * osd_architecture/osd-support.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="support-reviewing-an-access-request-from-an-email-notification_{context}"]
+= Reviewing an access request from an email notification
+
+.Prerequisites
+
+ifndef::openshift-rosa,openshift-dedicated[]
+* You have access to the cluster as a user with the `cluster-admin` role.
+endif::openshift-rosa,openshift-dedicated[]
+
+
+.Procedure
+
+. Click the link within the email to bring you to the {hybrid-console-second}.
+
+. Select *Open* for the *Pending* state.
```

Comment:
"Access Request Details" modal is already whenever you come from an email notification link

Reply:
removed step 2, thanks